scripts/publish.sh

```diff
@@ -3,8 +3,7 @@
 set -euo pipefail
 
 npm publish --cwd ./packages/blockfrost && \
-npm publish --cwd ./packages/cardano-graphql-db-sync && \
-npm publish --cwd ./packages/cardano-graphql-sps-provider && \
+npm publish --cwd ./packages/cardano-graphql && \
```

Comment:
Sorry @mkazlauskas , I probably wasn't clear and it's not obvious, but the intent is to have two packages since the first gen API isn't being brought forward. It's better to think of the two as separate services:

1. `cardano-graphql-db-sync` :point_right:  Gen 1
2. `cardano-graphql` :point_right: Gen 2 with our new stake pool search API